ci/jenkins/test.sh

```diff
@@ -270,22 +264,16 @@ function check_dhcp_status {
 
 function wait_for_antrea_windows_pods_ready {
     kubectl apply -f "${WORKDIR}/antrea.yml"
-    if [[ "${PROXY_ALL}" == false && ${TESTCASE} =~ "windows-e2e" ]]; then
-        kubectl apply -f "${WORKDIR}/kube-proxy-${WINDOWS_YAML_SUFFIX}.yml"
-    fi
-    kubectl apply -f "${WORKDIR}/antrea-${WINDOWS_YAML_SUFFIX}.yml"
 
     # Set trap to catch any errors from subsequent commands
     trap collect_windows_network_info_and_logs ERR
 
+    kubectl apply -f "${WORKDIR}/${WINDOWS_YAML_SUFFIX}.yml"
```

Comment:
Given that you've added the `antrea-` prefix in WINDOWS_YAML_SUFFIX, maybe rename it to `WINDOWS_YAML_NAME`? Or you can keep it the original way without the `antrea-` prefix.